styles/components/_modalWindow.scss

```diff
@@ -38,13 +44,16 @@
     font-weight: 500;
     font-size: 1.75rem;
     line-height: 2.125rem;
+    text-align: justify;
```

Comment:
Avoid text justification

Reply:
I think in this case it looks better, the heading creates clear "box" together with other elements.